admin/build/createpackage

```diff
@@ -1,85 +0,0 @@
-#!/bin/bash
```

Comment:
Will you make that change and release it? As you know, there is no RC
branch with cherry-picking. It will be easier just to do all of this
together now.

On Mon, Aug 17, 2015 at 10:39 AM, Paul Draper <notifications@github.com>
wrote:

> In admin/build/createpackage
> <https://github.com/lucidsoftware/piezo/pull/32#discussion_r37207672>:
>
> > @@ -1,85 +0,0 @@
> > -#!/bin/bash
>
> It works. But it's configuration was moved to the more standard
> /etc/piezo-admin/conf.
>
> That will have to be changed in Lucid's Chef configuration. I have not
> done that.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lucidsoftware/piezo/pull/32/files#r37207672>.
>